Java 16 | Vaadin 20.0.5 | Linux (e.g. Ubuntu 21.04) - production build fails at build-frontend goal

### Description of the bug / feature
When running the production build of a clean flow application a downstream js library fails to build the frontend on linux environments including CI environments using lnux based docker images to build.

Hints: 
* The build is working perfectly fine on my Windows 10 environment when using the same settings.
* The build started to fail on my CI environment yesterday (05.08.2021) in the afternoon. An earlier build was working fine. Due to some caching of artifacts the issue could have been caused earlier.

### Minimal reproducible example
Download a Vaadin starter from start.vaadin.com with the following settings:
* Vaadin 20
* Java 16
* Everything else default
Build the application on a linux environment using java 16 using the production profile (e.g. "mvn package -Pproduction")
### Expected behavior
The build should be successful.
### Actual behavior
Getting an exception during build-frontend:

> 
> [ERROR] Error: Cannot find module 'workbox-build/build/options/schema/webpack-generate-sw'
> [ERROR]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
> [ERROR]     at Function.Module._load (internal/modules/cjs/loader.js:508:25)
> [ERROR]     at Module.require (internal/modules/cjs/loader.js:637:17)
> [ERROR]     at require (/home/steven.ladewig/Downloads/my-app/node_modules/.pnpm/v8-compile-cache@2.0.3/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
> [ERROR]     at Object.<anonymous> (/home/steven.ladewig/Downloads/my-app/node_modules/.pnpm/workbox-webpack-plugin@6.1.0_webpack@4.46.0/node_modules/workbox-webpack-plugin/build/generate-sw.js:20:33)
> [ERROR]     at Module._compile (/home/steven.ladewig/Downloads/my-app/node_modules/.pnpm/v8-compile-cache@2.0.3/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
> [ERROR]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
> [ERROR]     at Module.load (internal/modules/cjs/loader.js:600:32)
> [ERROR]     at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
> [ERROR]     at Function.Module._load (internal/modules/cjs/loader.js:531:3)
> [ERROR]     at Module.require (internal/modules/cjs/loader.js:637:17)
> [ERROR]     at require (/home/steven.ladewig/Downloads/my-app/node_modules/.pnpm/v8-compile-cache@2.0.3/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
> [ERROR]     at Object.<anonymous> (/home/steven.ladewig/Downloads/my-app/node_modules/.pnpm/workbox-webpack-plugin@6.1.0_webpack@4.46.0/node_modules/workbox-webpack-plugin/build/index.js:10:20)
> [ERROR]     at Module._compile (/home/steven.ladewig/Downloads/my-app/node_modules/.pnpm/v8-compile-cache@2.0.3/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
> [ERROR]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
> [ERROR]     at Module.load (internal/modules/cjs/loader.js:600:32)
> [ERROR]     at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
> [ERROR]     at Function.Module._load (internal/modules/cjs/loader.js:531:3)
> [ERROR]     at Module.require (internal/modules/cjs/loader.js:637:17)
> [ERROR]     at require (/home/steven.ladewig/Downloads/my-app/node_modules/.pnpm/v8-compile-cache@2.0.3/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
> [ERROR]     at Object.<anonymous> (/home/steven.ladewig/Downloads/my-app/webpack.generated.js:10:28)
> [ERROR]     at Module._compile (/home/steven.ladewig/Downloads/my-app/node_modules/.pnpm/v8-compile-cache@2.0.3/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
> [ERROR]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
> [ERROR]     at Module.load (internal/modules/cjs/loader.js:600:32)
> [ERROR]     at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
> [ERROR]     at Function.Module._load (internal/modules/cjs/loader.js:531:3)
> [ERROR]     at Module.require (internal/modules/cjs/loader.js:637:17)
> [ERROR]     at require (/home/steven.ladewig/Downloads/my-app/node_modules/.pnpm/v8-compile-cache@2.0.3/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
> [ERROR]     at Object.<anonymous> (/home/steven.ladewig/Downloads/my-app/webpack.config.js:10:22)
> [ERROR]     at Module._compile (/home/steven.ladewig/Downloads/my-app/node_modules/.pnpm/v8-compile-cache@2.0.3/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
> [ERROR] -> [Help 1]

### Versions:
    - Vaadin / Flow version: 20.0.5
    - Java version: openjdk 16.0.1 | adoptopenjdk 16
    - OS version: Ubuntu 21.04 OR Docker image: maven:3-openjdk-16-slim or maven:3-openjdk-16 or maven:3-adoptopenjdk-16

--- a/flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java
+++ b/flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java
@@ -383,7 +383,7 @@ public abstract class NodeUpdater implements FallibleCommand {
         defaults.put("fork-ts-checker-webpack-plugin", "6.2.1");
 
         defaults.put("webpack", "4.46.0");
-        defaults.put("webpack-cli", "3.3.11");
+        defaults.put("webpack-cli", "3.3.12");
         defaults.put("webpack-dev-server", "3.11.0");
         defaults.put("compression-webpack-plugin", "4.0.1");
         defaults.put("extra-watch-webpack-plugin", "1.0.3");
@@ -393,7 +393,7 @@ public abstract class NodeUpdater implements FallibleCommand {
         defaults.put("lit-css-loader", "0.1.0");
         defaults.put("file-loader", "6.2.0");
         defaults.put("loader-utils", "2.0.0");
-        final String WORKBOX_VERSION = "6.1.0";
+        final String WORKBOX_VERSION = "6.2.0";
         defaults.put("workbox-webpack-plugin", WORKBOX_VERSION);
         defaults.put("workbox-core", WORKBOX_VERSION);
         defaults.put("workbox-precaching", WORKBOX_VERSION);

